content/en/blog/2025/otel-rocks.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "OTel Sucks (But Also Rocks!): Real Stories from the Trenches"
+linkTitle: OTel Sucks (But Also Rocks!)
+date: 2025-04-03
+author: '[Juraci Paixão Kröhling](https://github.com/jpkrohling) (OllyGarden)'
+cSpell:ignore: Olly Pismo Liatrio Moessis Kovalenko
+---
+
+OpenTelemetry (OTel) is often touted as the future of observability, promising vendor neutrality and comprehensive data collection. But what's the reality for those who use it daily? We sat down with several engineers and SREs to get their unfiltered thoughts on OTel. The result? A candid conversation about the good, the bad, and the sometimes frustrating aspects of working with OTel.
+
+In preparation for the KubeCon talk "[OTel Sucks (But Also Rocks!)](https://sched.co/1txHm)", Juraci spoke with community members and gathered a wealth of valuable insights. Due to time constraints, not all the material could be included in the presentation, so this is an attempt to do justice to the community's contributions.
+
+## Where OTel Falls Short
```

Comment:
I know the post covers a talk, but I'd like to know what we're doing, as a project, to address the pains. All complex projects have great and less great parts, so that's hardly surprising. Can we enrich what came out of the talk with our own stance as a project?

Reply:
The talk didn't happen yet, so nothing came out of it yet :-) What I can say right now is that the points mentioned are known to us all and we addressed them (semconv stability) or are addressing them (other semconv stability, collector stablity).

Reply:
Do you think it'd be valuable to add a note on this somewhere? That we're on it?

Reply:
Not sure: I think I prefer a honest "we are listening" tone. There are times where we just need to listen, and I think this is one of them. Perhaps we could have a "OTel Rocks" next year, showing how we progressed on the topics brought this year?

Reply:
Not sure: I think I prefer a honest "we are listening" tone. There are times where we just need to listen, and I think this is one of them. Perhaps we could have a "OTel Rocks" next year, showing how we progressed on the topics brought this year?